puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/3.10.groovy

```diff
@@ -1,6 +1,10 @@
 pipeline {
     agent none
 
+    environment {
+        version = "1.20"
```

Comment:
should this be `foreman_version` or `packaging_version`? as Katellos version is 3.10 here (applies to all katello files)